src/DatePicker/components/Month.vue

```diff
@@ -1,9 +1,12 @@
 <template>
-  <div ref="datepickerMonth" class="vhd__datepicker__month" @mouseenter="enterMonth($event)">
+  <div ref="datepickerMonth" class="vhd__datepicker__month" :class="{ 'vhd__datepicker__month--with-week-numnbers': showWeekNumbers }" @mouseenter="enterMonth($event)">
     <p class="vhd__datepicker__month-name">
       {{ monthName }}
     </p>
     <week-row v-bind="$props" />
+    <div v-if="showWeekNumbers" class="vhd__datepicker__weeknumbers">
+      <div class="vhd__datepicker__weeknumbers__number"  v-for="(day, index) in weekNumbers" :key="index" v-html="getIsoWeek(day.date)"></div>
```

Comment:
Please don't use index as a key, because does not allows Vue to properly identify this div among others to update the DOM